packages/chain-adapters/src/bitcoin/BitcoinChainAdapter.ts

```diff
@@ -378,10 +378,10 @@ export class ChainAdapter implements IChainAdapter<ChainTypes.Bitcoin> {
     const { xpub } = await this.getPubKey(wallet, bip32Params, scriptType)
     const account = await this.getAccount(xpub)
     const addresses = (account.chainSpecific.addresses ?? []).map((address) => address.pubkey)
-    const id = `${toRootDerivationPath(bip32Params)}/${scriptType}`
+    const subscriptionId = `${toRootDerivationPath(bip32Params)}/${scriptType}`
```

Comment:
My PR changes this line (scriptType to accountType). so either live it alone and use `id` as the variable name or someone will have to resolve a conflict.